test/IntegrationTests/MongoDbTests.cs

```diff
@@ -14,10 +14,8 @@
 // limitations under the License.
 // </copyright>
 
+#if !NETFRAMEWORK
```

Comment:
Can you please document in `config.md` that MongoDB instrumentation is not supported on .NET Framework?

Reply:
@RassK could you please check if this is correct 17b7df26